locales/templates/app.pot

```diff
@@ -1556,77 +1556,617 @@ msgid ""
+msgid "Strategy"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_7"
+msgid "Adventure"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_8"
+msgid "Action"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_9"
+msgid "Arcade"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_10"
+msgid "Puzzle & Logic"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_11"
+msgid "Sports & Racing"
+msgstr ""
+
+msgctxt "gaming_vertical_genre_12"
+msgid "Girls"
+msgstr ""
+
+#. Home 2.0
+#. Onboarding for existing users to upgrage to Firefox Lite 2.0. %1$s will be
+#. replaced with the name of the app (e.g. Firefox Lite)
+#, c-format
+msgctxt "onboarding_2_introduction_title"
+msgid "Brand-new %1$s"
+msgstr ""
+
+msgctxt "onboarding_2_introduction_description"
+msgid ""
+"Remain Fast and Lightweight. We added smart shopping search and content "
+"to make internet browsing more fun and enjoyable!"
+msgstr ""
+
+#. Onboarding for new users to introduce Firefox Lite 2.0
+#. Introduce Content & Services
+msgctxt "onboarding_2_content_services_news_shopping_games"
+msgid ""
+"Tap to read the trendy news, find good deals or play the hottest free "
+"games to have fun!"
+msgstr ""
+
+msgctxt "onboarding_2_content_services_news_games"
+msgid "Tap to read the trendy news or play the hottest free games to have fun!"
+msgstr ""
+
+msgctxt "onboarding_2_content_services_games"
+msgid "Tap to play the hottest free games to have fun!"
+msgstr ""
+
+#. Introduce Smart Shopping Search
+msgctxt "onboarding_2_smart_shopping_search_title"
+msgid "Smart Shopping Search"
+msgstr ""
+
+msgctxt "onboarding_2_smart_shopping_search_description"
+msgid "Comparing prices across different online stores is super easy, try it now!"
+msgstr ""
+
+msgctxt "home_search_bar_text"
+msgid "Search..."
+msgstr ""
+
+#. Content & Services
+#. Title of Content & Services
+msgctxt "home_content_services"
+msgid "Content & Services"
+msgstr ""
+
+msgctxt "home_discovery"
+msgid "Discovery"
+msgstr ""
+
+#. Actions in context menu. Users can long press an item then show the context
+#. menu to select an action
+msgctxt "content_home_context_menu_add_to_home_screen"
+msgid "Add to Home screen"
+msgstr ""
+
+msgctxt "content_home_context_menu_share_link"
+msgid "Share link"
 msgstr ""
 
-#. Message shown in Home Screen with an icon. Users can choose either they
-#. want this feature or not. We'll evaluate if we want to implement this
-#. feature base on the users' feedback.
-msgctxt "exp_survey_vpn"
+#. Error message. E.g. when open a service and server is not working then show
+#. error message to users
+msgctxt "content_home_error_message"
 msgid ""
-"We plan to integrate “VPN service” to get you more freedom and more "
-"privacy when browsing.\n"
+"Oops!\n"
+"Something went wrong"
+msgstr ""
+
+#. Private mode home 2.0
+#. Title of Private mode
+msgctxt "private_home_title"
+msgid "Advanced Private Browsing"
+msgstr ""
+
+#. Description of Private mode
+msgctxt "private_home_description"
+msgid ""
```

Comment:
String ID needs to be updated

Reply:
Could you tell more about why the id needs to be updated? Anything wrong??

Reply:
My bad here too, sorry